src/drive/web/modules/drive/Container.jsx

```diff
@@ -64,6 +64,7 @@ const isAnyFileReferencedByAlbum = files => {
 }
 
 const mapStateToProps = state => ({
+  files: state.view.files,
```

Comment:
I'm afraid don't understand this change.

This prop is passed to the `Container` component, which is then rendered by the FileExplorer. But the [FIleExplorer already injects a `files` prop](https://github.com/y-lohse/cozy-drive/blob/photos-relations/src/drive/web/modules/navigation/FileExplorer.jsx#L84).

So now we're injecting the prop twice, but one time using directly the state value, and one time using the `getVisibleFiles` selector.

Do we really want to inject the same prop twice? And why does injecting it directly with   `state.view.files` work, but now with `getVisibleFiles`?

Reply:
Oh nice catch! 
In fact, we display only the filename if we don't have a `displayedFolder`https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/navigation/duck/reducer.js#L372 

Since we're in the `recent view`, we don't have any displayedFolder. 

We should probably edit this `getVisibilesFiles` selector ! 👍

Reply:
Aaah yes, I didn't go all the way down to the `getFilePath ` selector, this does explain the problem. So yeah I also think it would be best to fix this problem in the selectors.

Reply:
oh, just saw your comments (I was already on it). In the end, I did this: https://github.com/cozy/cozy-drive/pull/1366/commits/cf88e57c149943b77d32870226cca69f7e4b8228

It feels strange and misleading that we have two different information in `path` attribute. In case of a file Recent view, we have the path to the parent dir and for other files, we have the full path. I am afraid to break stuff so I don't want to go too far here but I don't really like this :-)